esx_service/cli/vmdkops_admin.py

```diff
@@ -619,24 +623,15 @@ def get_vmdk_size_info(path):
     Get the capacity and used space for a given VMDK given its absolute path.
     Values are returned as strings in human readable form (e.g. 10.00MB)
 
-    Currently this data is retrieved via a call to vmkfstools.
-    The output being parsed looks like the following:
-
-    Capacity bytes: 209715200
-    Used bytes: 27262976
-    Unshared bytes: 27262976
+    Using get_vol_info api from volume kv. The info returned by this
+    api is in human readable form
     """
     try:
-        cmd = "vmkfstools --extendedstatinfo {0}".format(path).split()
-        output = subprocess.check_output(cmd)
-        result = output.decode('utf-8')
-        lines = result.split('\n')
-        capacity_in_bytes = lines[0].split()[2]
-        used_in_bytes = lines[1].split()[2]
-        return {'capacity': human_readable(int(capacity_in_bytes)),
```

Comment:
can you please remove```human_readable``` method? as there will not be any consumer with your proposed changes.